specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -5202,6 +5202,270 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/services/{serviceName}": {
```

Comment:
Since services API is generic API, we will try to incorporate the genericness in swagger definitions as well. 
Let us create an enum, service type which can take the values DataTransfer and SqlDedicatedGateway (in future apiVersions we will add MVBuilder and Janus Graph as well)